modules/fedimint-mint-client/src/lib.rs

```diff
@@ -341,7 +341,11 @@ pub struct MintOperationMeta {
 #[serde(rename_all = "snake_case")]
 pub enum MintOperationMetaVariant {
     Reissuance {
-        out_point: OutPoint,
+        // Deleted in 0.3.0:
+        // # out_point: OutPoint,
```

Comment:
second thought: want to keep it around to still at least validate tx acceptance of old receives